compiler-rt/lib/sanitizer_common/sanitizer_libignore.h

```diff
@@ -54,24 +54,51 @@ class LibIgnore {
     char *name;
     char *real_name;  // target of symlink
     bool loaded;
+    uptr ignored_code_range_id;
   };
 
   struct LibCodeRange {
     uptr begin;
     uptr end;
   };
 
+  // Marks a range as loaded by utilizing the least significant bit of the code
+  // range. Assumes the start of the code range is 2-byte aligned.
+  struct LibLoadedCodeRange {
+    uptr begin() const { return begin_ << 1; }
```

Comment:
Please remove begin(uptr) and replace with LibLoadedCodeRange(begin, end)

Reply:
Makes sense yes, not needed I think following your suggested simplification.